mocap_qualisys/launch/qualisys.launch

```diff
@@ -1,11 +1,14 @@
 <launch>
+  <arg name="server_address" default="11.0.0.10"/>
+  <arg name="server_base_port" default="22222"/>
+  <arg name="frame_rate" default="100"/>
   <node pkg="mocap_qualisys"
     type="mocap_qualisys_node"
     name="qualisys"
     output="screen">
-    <param name="server_address" value="11.0.0.10"/>
-    <param name="server_base_port" value="22222"/>
-    <param name="frame_rate" value="0"/>
+    <param name="server_address" value="$(arg server_address)"/>
+    <param name="server_base_port" value="$(arg server_base_port)"/>
+    <param name="frame_rate" value="$(arg frame_rate)"/>
     <param name="max_accel" value="10.0"/>
```

Comment:
I like the idea of breaking out the parameters to arguments, but it should be done to all parameters at once to be consistent.